lib/arkana/models/type.rb

```diff
@@ -20,4 +20,8 @@ def self.new(string_value:)
         STRING
     end
   end
+
+  def self.default_string
```

Comment:
Could you also rename this property and add a documentation touch? 🙇 

```suggestion
  # The default type to use when we shouldn't be inferring types.
  def self.default_type
```